app/src/main/java/com/github/se/signify/ui/screens/challenge/ChallengeHistoryScreen.kt

```diff
@@ -213,32 +220,36 @@ fun determineWinner(
     player1: String,
     player2: String,
     player1Result: Double,
-    player2Result: Double
+    player2Result: Double,
+    drawText: String
 ): String {
+
   return when (mode) {
     ChallengeMode.SPRINT.toString() -> {
       when {
         player1Result > player2Result -> player1
         player2Result > player1Result -> player2
-        else -> "Draw"
+        else -> drawText
       }
     }
     else -> {
       when {
         player1Result < player2Result -> player1
         player2Result < player1Result -> player2
-        else -> "Draw"
+        else -> drawText
       }
     }
   }
 }
 
 @Composable
 fun PlayerScoreText(player: String, result: Double, mode: String) {
+  val wordsText = stringResource(R.string.words)
+
   val scoreText =
       when (mode) {
         ChallengeMode.CHRONO.toString() -> "$player Score: $result s"
-        else -> "$player Score: $result words"
+        else -> "$player Score: $result $wordsText"
```

Comment:
There is hardocded string here for score even if it's in french "Score"

Reply:
Done, ty